config/jobs/kubernetes-sigs/sig-windows/sig-windows-config.yaml

```diff
@@ -70,10 +70,10 @@ presubmits:
         - "--acsengine-admin-username=azureuser"
         - "--acsengine-admin-password=AdminPassw0rd"
         - "--acsengine-creds=$AZURE_CREDENTIALS"
-        - "--acsengine-download-url=https://github.com/Azure/aks-engine/releases/download/v0.31.2/aks-engine-v0.31.2-linux-amd64.tar.gz"
+        - "--acsengine-download-url=https://aka.ms/aks-engine/aks-engine-k8s-e2e.tar.gz"
```

Comment:
is this always downloading the latest tar?
isn't it better to pin to certain tag?

Reply:
Basically it points to the latest release we can use. However, since we are testing master k8s, there are situations where aks-engine won't start kubelet / other components properly: i.e some weeks ago --allow-privileged flag was removed. For situations like this, we might need to quickly create a custom aks-engine build that will work so we don't have too much downtime on tests. It's easier and faster to just update the short link than to change the config.

I'll actually update the description with this line of reasoning for clarity.
I'll also have to update kubetest to post the version of aks-engine it downloaded ( i.e run aks-engine version )